app/assets/stylesheets/datapicker.scss

```diff
@@ -0,0 +1,9 @@
+/*!

```

Comment:
Third-party components add please in "/vendor" folder. The javascript file of this component too.